packages/block-library/src/post-comments-form/form.js

```diff
@@ -45,6 +50,7 @@ const CommentsFormPlaceholder = () => {
 						) }
 						label={ __( 'Post Comment' ) }
 						value={ __( 'Post Comment' ) }
+						aria-disabled="true"
```

Comment:
Same here - can we use `disabled` instead?

Reply:
It would remove the button from the tab order. Take a look at the Button component in wordpress/components. I followed a similar approach to disable the button in an accessible way. Might need to visually adjust though.

Reply:
I think it's fine for the button to not in the tab order, as the form doesn't do anything. It seems like the intention was for the form to not be interacted with on the editor, so marking it as `disabled` would keep the same behavior. 

That said, I see the argument for marking it as disabled with `aria-disabled` over `disabled` so that you _can_ interact with it. I'm honestly fine with it either way. I feel comfortable approving this PR with `disabled` since it preserves the same behavior as trunk while also removing the inaccessible `inert`. I'm also very aware I could be missing something here, so let me know what you think!

Reply:
Yes, it should be fine.

Reply:
Sorry, misread your comment. I think it is best to keep `aria-disabled` that way users can still get a feel for the form layout via the Tab/Shift+Tab shortcuts. Users will still get the idea that the form controls are disabled but nothing will happen when they try to use them.

The `textarea` now uses `readOnly` attribute to prevent text entry.

Reply:
I'd agree to keep `aria-disabled`. 
The purpose of the editor is to build a page and show a preview. If the buttons is presented visually to users, then it also needs to be perceivable by non sighted users. As Alex mentioned, `disabled` would remove the button from the tab sequence, making it less discovereable.
Also, `aria-disabled` + `nooping` a control is a common pattern we use since years in the editor to keep disabled controls discoverable.

Reply:
I can see why using `aria-disabled` + `nooping` is a better, more equivalent experience. Thanks for the explanation, @alexstine and @afercia!